CASMTRIAGE-6646 Deterministic SNMP Settings (#4723)
Cray-HPE/docs-csm: operations/network/management_network/snmp_exporter_configs.md

In order to provide data to the Grafana SNMP dashboards, the SNMP Exporter must be configured with a list of management network switches to scrape metrics from.

> ***NOTE*** All variables used within this page depend on the `/etc/environment` setup done in [Pre-installation](../../../install/pre-installation.md).

> 1. (`pit#`) Obtain the list of switch names and their IP addresses in a format that be used with copy/paste into the next command.

Reviewer: ```suggestion
1. (`pit#`) Obtain the list of switch names and their IP addresses in a format to use with copy/paste in the next command.
```

Reviewer: Can we be more specific on which format will work with copy/paste?

    ```bash
    jq '.Networks.NMN.ExtraProperties.Subnets | .[] | select(.FullName=="NMN Management Network Infrastructure").IPReservations | map( {name: .Name, target: .IPAddress })' \
    "${PITDATA}/prep/${SYSTEM_NAME}/sls_input_file.json"
    ```

    Expected output looks similar to the following:

    ```json
    [
      {
        "name": "sw-spine-001",
        "target": "10.252.0.2"
      },
      {
        "name": "sw-spine-002",
        "target": "10.252.0.3"
      },
      {
        "name": "sw-leaf-bmc-001",
        "target": "10.252.0.4"
      }
    ]
    ```

> 1. (`pit#`) Update `customizations.yaml` with the list of switches, copy and paste the output from the previous command

Reviewer: ```suggestion
1. (`pit#`) Update `customizations.yaml` with the list of switches by copying and pasting the output from the previous command.
```
